config/routes.rb

```diff
@@ -7,6 +7,7 @@
 
   namespace :admin, path: "monologue" do
     get "/" => "posts#index", as:  "" # responds to admin_url and admin_path
+    get "/page/:page", to:  "posts#index", as:  "posts_page"
```

Comment:
`admin_posts_page_path` I wanted to keep the same format ( `posts_page_path` ) that already exists in the non-admin section. Here's the new route and helper:

![screen shot 2015-02-13 at 9 21 46 am](https://cloud.githubusercontent.com/assets/5491763/6191891/ea53d758-b361-11e4-84bf-e29807787b55.png)